tools/prepare-web-types.mjs

```diff
@@ -0,0 +1,11 @@
+import {readFile, writeFile} from 'fs/promises';
+
+const pluginsPath = new URL('../dist/ng-event-plugins/', import.meta.url);
+const webTypesPath = new URL('./web-types.json', pluginsPath);
+const webTypes = await readFile(webTypesPath, {
+    encoding: 'utf8',
+});
+
+const newWebTypes = webTypes.replaceAll(/"\.\/src\/([^"]+)\.ts"/g, (_, matched) => `"./${matched}.d.ts"`);
+
+await writeFile(webTypesPath, newWebTypes);
```

Comment:
why do we need that?

Reply:
All paths in web-types.json are broken after the build. This script will fix that.